### PR TITLE
Experimental support for double buffered fb

### PR DESCRIPTION
**Pull request by bschaef14**
Adds optional suffix to rawfb string that will enable x11vnc to map the virtual
size specified, not just screen size. x11vnc will then track yoffset of fb as it
scans for changes as this specifies which part of the fb is actually being
displayed.

### CODE REVIEW

**Comment by bschaef14** [comment]
Addresses #106

File: README [hunk 1]
```diff
@@ -16858,22 +16858,33 @@ er
                        discussion below where the framebuffer is taken as that
                        of another remote VNC server.
 
-                       Optional suffixes are ":R/G/B" and "+O" to specify
-                       red, green, and blue masks (in hex) and an offset into
-                       the memory object.  If the masks are not provided x11vnc
-                       guesses them based on the bpp (if the colors look wrong,
-                       you need to provide the masks.)
+                       Optional suffixes are ":R/G/B", "+O", and "#WxH" to
```
[the remark this answers is not in the document]
Comment (bschaef14) [reply README]: yes, fixed.

File: src/screen.c [hunk 2]
```diff
@@ -109,6 +109,7 @@ static int choose_delay(double dt);
 int rawfb_reset = -1;
 int rawfb_dev_video = 0;
 int rawfb_vnc_reflect = 0;
+int rawfb_db = 0;
```
[the remark this answers is not in the document]
Comment (bschaef14) [reply src/screen.c]: Apologies, I had some difficulty myself with variable names throughout :)

[the remark this answers is not in the document]
Comment (bk138) [reply src/screen.c]: yeah, the existing code is quite old-style C

File: src/linuxfb.c [hunk 3]
```diff
@@ -56,7 +56,7 @@ void console_pointer_command(int mask, int x, int y, rfbClientPtr client);
 int rawfb_get_offset(int *fd);
 void linux_dev_fb_msg(char *);
 
-/// Returns offset into fb memory based on yoffset.
+// Returns offset into fb memory based on yoffset.
```
Comment (bk138) [inline src/linuxfb.c]: Still C++ style here. Please use `/* comment */`

**Comment by bk138** [comment]
@bschaef14 two minor things:  my remark about `//` vs `/* */` and when you're done, please squash the changes into one commit. Then we're ready!

**Comment by bk138** [comment]
Very neato! Thanks!

**Pull request merged**